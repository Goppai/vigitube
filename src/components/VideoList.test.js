import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { VideoList } from './'

describe('<VideoList/>', () => {
  const wrapper = shallow(<VideoList/>)

  it('renders', () => {
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
