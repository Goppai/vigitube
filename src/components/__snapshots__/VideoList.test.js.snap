// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoList/> renders 1`] = `
<Col
  className="video-list"
  lg={7}
  xs={24}
>
  <List
    bordered={false}
    dataSource={Array []}
    itemLayout="vertical"
    loadMore={
      <Button
        className="load-more-btn"
        ghost={false}
        loading={false}
        onClick={undefined}
        prefixCls="ant-btn"
        type="primary"
      >
        Load More
      </Button>
    }
    loading={false}
    pagination={false}
    prefixCls="ant-list"
    renderItem={[Function]}
    split={true}
  />
</Col>
`;
